Sources/Plasma/PubUtilLib/plGLight/plLightInfo.h

```diff
@@ -276,16 +278,16 @@ class plLimitedDirLightInfo : public plDirectionalLightInfo
 
     plParallelIsect*            fParPlanes;
 
-    void                IMakeIsect() override;
-    plVolumeIsect*      IGetIsect() override { return fParPlanes; }
+    void           IMakeIsect() override;
+    plVolumeIsect* IGetIsect() const override;
```

Comment:
Returning a pointer can be kept inline

Reply:
Unfortunately, all of the `IGetIsect()` methods return subclasses of `plVolumeIsect`, which is an incomplete type. I could add the include back, but 😢

Reply:
Ah, I see...  I was thinking that didn't matter, but I guess it would need to know the inheritance relationship for that to work :/